### PR TITLE
2.2.0 - Add actions and workflows for python - and comment out code cov due to misbehavior

### PR DESCRIPTION
**Pull request by djorgensendk**

### CODE REVIEW

File: .github/actions/databricks-unit-test/action.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: 'PySpark execution'
```
Comment (Renetnielsen) [inline .github/actions/databricks-unit-test/action.yml:1]: License header needs to be added to all files that allows comments. (Not sure if the .py does)

File: .github/workflows/dotnet-solution-ci.yml [hunk 2]
```diff
@@ -130,9 +130,9 @@ jobs:
...
           dotnet test ${{ inputs.SOLUTION_FILE_PATH }} --no-build --configuration ${{ env.BUILD_CONFIGURATION }} --verbosity normal --collect:"XPlat Code Coverage" -- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=opencover --output ${{ github.workspace }}/output
 
-      - name: Upload coverage
-        uses: codecov/codecov-action@v1
-        with:
-          flags: unittests
-          fail_ci_if_error: true
-          verbose: true
\ No newline at end of file
+      # - name: Upload coverage
```
Comment (Renetnielsen) [inline .github/workflows/dotnet-solution-ci.yml:133]: Remove this code, using git we have it in history if needed.
Its prettier :)

**Review by Renetnielsen — CHANGES_REQUESTED** [review]

File: .github/actions/databricks-unit-test/action.yml (hunk 1 above)
Comment (djorgensendk) [reply .github/actions/databricks-unit-test/action.yml:15]: added on all files

File: .github/workflows/dotnet-solution-ci.yml (hunk 2 above)
Comment (djorgensendk) [reply .github/workflows/dotnet-solution-ci.yml]: removed

**Review by Renetnielsen — APPROVED** [review]

**Pull request merged**